flow/Makefile

```diff
@@ -158,9 +166,37 @@ ifeq (, $(strip $(NPROC)))
 endif
 export NUM_CORES := $(NPROC)
 
-OPENROAD_CMD = openroad -no_init -exit
-OPENROAD_NO_EXIT_CMD = $(patsubst -exit,,$(OPENROAD_CMD))
+#-------------------------------------------------------------------------------
+# setup all commands used within this flow
+TIME_CMD = /usr/bin/time -f "%Eelapsed %PCPU %MmemKB"
+TIME_TEST = $(shell $(TIME_CMD) echo foo 2>/dev/null)
+ifeq (, $(strip $(TIME_TEST)))
+  TIME_CMD = /usr/bin/time
+endif
 
+# The following determine the executable location for each tool used by this flow.
+# Priority is given to 
+#       1 user include path to the executable prior to running make
+#       2 user explicity set with variable in Makefile or command line
+OPENROAD_CMD            ?= $(shell command -v openroad)
+ifeq ($(OPENROAD_CMD),)
+  OPENROAD_CMD           = $(abspath $(OPENROAD_HOME)/../tools/build/OpenROAD/src/openroad) -no_init -exit
+else
+  OPENROAD_CMD           = $(shell command -v openroad) -no_init -exit
+endif
+OPENROAD_NO_EXIT_CMD     = $(firstword $(OPENROAD_CMD)) -no_exit
```

Comment:
`-no_exit` is not an option.
Maybe simplify to:
`OPENROAD_NO_EXIT_CMD            ?= $(shell command -v openroad)`
`ifeq ($(OPENROAD_NO_EXIT_CMD),)`
`  OPENROAD_NO_EXIT_CMD           = $(abspath $(OPENROAD_HOME)/../tools/build/OpenROAD/src/openroad)`
`endif`
`OPENROAD_NO_EXIT_CMD += -no_init`
`OPENROAD_CMD = $(OPENROAD_NO_EXIT_CMD) -exit`

This way adding new flags to openroad doesn't require editing in multiple lines